hyperspace.py

```diff
@@ -139,7 +139,10 @@ def do_newuser(args):
     if db.usr_getby_username(args.username)[0]:
         errx(f'cannot create duplicate user "{args.username}"')
     else:
-        db.usr_ins((args.username, do_bcrypt_hash(args, get=True), 0, 0))
+        db.usr_ins((args.username, do_bcrypt_hash(args, get=True),
+                    0, args.studentid or 0))
```

Comment:
interesting usage of or here but it works

Reply:
yeah, js behaves similarly, AND returns the first falsey value and OR returns the first truthy value